packages/manager/modules/exchange/src/external-contact/external-contact.html

```diff
@@ -3,220 +3,117 @@
     data-ng-controller="ExchangeTabExternalContactsCtrl as ctrl"
 >
     <div class="row">
-        <div class="col-md-9">
-            <form
-                class="form-inline d-md-flex justify-content-md-end mb-3"
-                name="searchExternalContactForm"
+        <div class="col-md-12">
```

Comment:
I think you can then rely on a single `<div>` element (`.row > .col-XX-12` === `<div>`).